### ISSUE TITLE
[Question]PreBuild Packages & development?

### ISSUE THREAD

**Issue by yoshimo** [iSECPartners/android-ssl-bypass#2]
Currently there seems to be a focus on killing ssl on IOS only,
the project description shows "will be frequently updated in the near future."
The last update was almost 8 months ago, is this project abandoned?
If not, some prebuild apks would be fine, similiar to the ios deb package

**Comment by nabla-c0d3** [comment]
Hi,
We're pretty close to releasing an Android tool that's exactly like the iOS tool and packaged as an apk. I'll let you know once it's released.

**Comment by yoshimo** [comment]
Does this also allow to see traffic caused by native android code or just classic java?

**Comment by nabla-c0d3** [comment]
Classic Java for now. Doing certificate pinning in native android code seems pretty elaborate... have you seen anyone doing it ?

**Comment by yoshimo** [comment]
No, i misunderstood a problem i had with an app that is full of native libraries.
It took some work to force it to use the proxy i was using for the test, so it a lot of traffic was lost before that

**Comment by nabla-c0d3** [comment]
It is live:
https://github.com/iSECPartners/Android-SSL-TrustKiller
